modes and stuff

PokeSvorlds opened the pull request:
-made role and gamemode lists nicer
-fixed a couple current lycanthropy bugs and seperated doomsayer and totem lycans
-made piper give full charmed list each night
-bug fixing
     -fixed some bodyguard bugs
     -fixed crow not seeing some roles
     -fixed kill teams a little
     -amne turning when dead
-default mode roles changed
-chaos mode roles changed
-drunkfire and orgy changed minorly
-gave belunga, evilvillage, and valentines higher max player counts

+revamped gamemode choosing to that of lykos
+added mad scientist, clone, lycan, time lord, turncoat roles
+added mad, lycan, rapidfire, noreveal, aleatoire modes

belguawhale committed:
fix harlot infinite visit

belguawhale committed:
Merge branch 'PokeSvorlds-master' of https://github.com/belguawhale/Discord-Werewolf into PokeSvorlds-master

PokeSvorlds committed:
Doubly mad (#38)

* Mad scientist + clone

+ mad mode
clone instead of cs in foolish

minor fixes
augur sees evil amnes, no retribution in wolf shaman totems, less "wtf"ing when people use pass

PokeSvorlds committed:
cleanup ig

PokeSvorlds committed:
Update bot.py

PokeSvorlds committed:
Gamemode changes and clone honing

*the rest of the stuff already exists in this branch but it's adding them again @_@

PokeSvorlds committed:
Gamemode changes, mostly chaos

*Chaos overhaul, especially at 8p+
*Less cs, more orgy in orgy
*Fixed valentines, default numbers

PokeSvorlds committed:
One more gamemode change for the road

*Earlier drunks in drunkfire

PokeSvorlds committed:
evil and val extensions

PokeSvorlds committed:
oops

PokeSvorlds committed:
Minor fixes

A little of everything, a fix for mad, one for clone, one for hunter and one to clean up a list

PokeSvorlds committed:
randium's favorite name

*updates role list again
*made amnesiac hunters/vgs doubly unable to retract
*heavier amnesiac blacklist
*really ugly fix to made mad gunners have one bullet
*fixed double bodyguards and bodyguarding mad kill
*added sharpshooters in drunkfire
+added lycan role
*gave doom lycan it's own other flag and message

PokeSvorlds committed:
noreveal

PokeSvorlds committed:
noreveal aleatoire 

Hopefully final changes but probably not

PokeSvorlds committed:
Delete settings.py.example

PokeSvorlds committed:
Rename settings.py to settings.py.example

PokeSvorlds committed:
fixed my clone rewrite and made times always change back

27979Raven commented:
Nice.

27979Raven commented:
I wish I knew how to code so I could really help unless one of you could teach me the ropes

PokeSvorlds committed:
clone !stats, better turncoat and passing

PokeSvorlds committed:
night shall wait for the ghosts

PokeSvorlds committed:
bodyguard not being harlot

PokeSvorlds committed:
neutrals dont help kill the evilvillage

belguawhale committed:
Merge branch 'master' into PokeSvorlds-master

PokeSvorlds committed:
better silencing, turncoat working?

turncoat was being very fickle so i turned it back to a older, slightly uglier system
silenced shamans dont hold up night, mystics are actually silenced, and more roles are told they're silenced when they are
doomsayer actually uses power up

PokeSvorlds committed:
join order lists and a mad fix

Took out sort_player for all relevant lists I could find
mad will skip in rapid and random - my python grammar is shit

PokeSvorlds committed:
unfucking mudkip totems

PokeSvorlds committed:
dead lycans can win and hunter cannot shoot after passing

PokeSvorlds committed:
wiki links, hidden amne roles, win con and role desc cleaning
